news/WildFly-s2i-19-released.adoc

```diff
@@ -0,0 +1,149 @@
+= WildFly 19 S2I image has been released on quay.io
+jfdenise
+2020-03-20
+:awestruct-tags: [wildfly, galleon]
+:awestruct-layout: blog
+:source-highlighter: coderay
+:encoding: utf-8
+:lang: en
+
+==  WildFly 19 S2I docker images
+
+WildFly s2i builder and runtime Docker images for WildFly 19 have been released on link:https://quay.io/organization/wildfly[quay.io/wildfly]
+
+Changes since link:https://wildfly.org/news/2019/10/07/WildFly-s2i-18-released/[last release]:
+
+* Optimized startup. The server starts much faster, the CLI script applied to the server configuration is now executed during server boot.
+* Support for clustering. JGroups is now part of the default server configuration. 
+* WildFly 19 Galleon decorator layer _web-clustering_ (support for Infinispan-based web session clustering) is fully usable in the openshift context. 
+This blog post highlights the clustering new features.
+* Default standalone.xml configuration file includes the Microprofile JWT (JSON Web Token) subsystem.
+* WildFly 19 defined new Galleon layers for Microprofile 3.3 (_microprofile-fault-tolerance_, _microprofile-jwt_ and _microprofile-openapi_) can be referenced from 
+_GALLEON_PROVISION_LAYERS_ env variable.
+
+For a complete documentation on how to use these images using s2i, OpenShift and Docker, 
+refer to link:https://github.com/wildfly/wildfly-s2i/blob/wf-19.0/README.md[README].
+
+== Adding imagestreams and template to OpenShift
+
+At some point the new images will be made available from OpenShift catalog and image repository. You can already use these images by adding them yourselves to your OpenShift cluster.
+
+* WildFly S2I builder image stream: 
+```
+oc create -f https://raw.githubusercontent.com/wildfly/wildfly-s2i/wf-19.0/imagestreams/wildfly-centos7.json
+```
+* WildFly runtime image stream: 
+```
+oc create -f https://raw.githubusercontent.com/wildfly/wildfly-s2i/wf-19.0/imagestreams/wildfly-runtime-centos7.json
+```
+* Chained build template: 
+```
+oc create -f https://raw.githubusercontent.com/wildfly/wildfly-s2i/wf-19.0/templates/wildfly-s2i-chained-build-template.yml
+```
+
+NB: If you import the image streams in your project, be sure to set _ImageStreams Namespace_ to your project namespace in the template. _openshift_ being the default namespace.
```

Comment:
I didn't include this in https://github.com/jfdenise/wildfly.org/pull/1 as it is more substantive than just a text edit.

Should this be 

"..., be sure to set the template parameter IMAGE_STREAM_NAMESPACE to your project namespace; _openshift_ being..."

I'm not sure what '_ImageStreams Namespace_ ' is.

Reply:
This is the string displayed for the argument in the console. I added the command line argument too.

Reply:
` IMAGE_STREAM_NAMESPACE` means the namespace where the user imported the image streams from the previous commands:

```
$ oc create -f https://raw.githubusercontent.com/wildfly/wildfly-s2i/wf-19.0/imagestreams/wildfly-centos7.json
```

which in that case will be whatever `oc project` currently is pointing to...

@jfdenise Maybe, it would be simpler to explicitly install the image streams in the `openshift`namespace with:

```
$ oc create -n openshift -f https://raw.githubusercontent.com/wildfly/wildfly-s2i/wf-19.0/imagestreams/wildfly-centos7.json
$ oc get -n openshift imagestream/wildfly
NAME      IMAGE REPOSITORY                                                            TAGS                                                  UPDATED
wildfly   default-route-openshift-image-registry.apps-crc.testing/openshift/wildfly   10.0,10.1,11.0,12.0,13.0,14.0,15.0,16.0 + 6 more...   17 seconds ago
```

Reply:
Installing in openshift means having the right to do it. All users can't, adding in your own namespace will work.